Resources/public/js/views/fields/ez-dateandtime-view.js

```diff
@@ -79,11 +79,22 @@ YUI.add('ez-dateandtime-view', function (Y) {
         _getDateObject: function () {
             var field = this.get('field');
 
-            if ( field && field.fieldValue ) {
+            if (this._isFieldEmpty()) {
+                return undefined;
+            } else {
                 return new Date(field.fieldValue.timestamp * 1000);
             }
-            return undefined;
-        }
+        },
+
+        _isFieldEmpty: function () {
+            var field = this.get('field');
+
+            if (field && field.fieldValue && field.fieldValue.timestamp !== null ) {
```

Comment:
can't you simplify that like in https://github.com/ezsystems/PlatformUIBundle/pull/501/files#diff-87487438348f321146e7acd2eefba122R311 ?